CMakeLists.txt

```diff
@@ -194,11 +194,14 @@ endif()
 
 if (WITH_IMAEVM)
 	list(APPEND REQFUNCS lsetxattr)
-	find_path(IMA_INCLUDE_DIR NAMES imaevm.h)
-	find_library(IMA_LIBRARY NAMES imaevm)
-	if (NOT (IMA_INCLUDE_DIR AND IMA_LIBRARY))
-		message(FATAL_ERROR "imaevm enabled but library not present")
-	endif()
+	find_path(IMA_INCLUDE_DIR NAMES imaevm.h REQUIRED)
+	find_library(IMA_LIBRARY NAMES imaevm REQUIRED)
```

Comment:
REQUIRED was only added in cmake 3.18, though. But without it, one has to use these awkward checks and custom error messages...

Reply:
Which makes me think if we shouldn't just bump our minimal cmake version?

Reply:
... aaaand done. 3.18 it is :smile: